src/ansys/sherlock/core/layer.py

```diff
@@ -244,16 +246,14 @@ def add_potting_region(
 
     @require_version(251)
     def update_potting_region(
-        self, project: str, update_potting_region_requests: list[PottingRegionUpdateData]
+        self, request: UpdatePottingRegionRequest
     ) -> list[SherlockCommonService_pb2.ReturnCode]:
         """Update one or more potting regions in a specific project.
 
         Parameters
         ----------
-        project: str
-            Name of the Sherlock project.
-        update_potting_region_requests: list[PottingRegionUpdateData]
-            List of data used to update potting regions.
+        request: UpdatePottingRegionRequest
+            Contains all the information needed to update 1 or multiple potting regions per project

```

Comment:
The sentence is missing the final period. And do we use numbers or rather "one" normally?

Reply:
It's better to use the word for a number in a sentence like this.